esp/src/src/ESPUtil.ts

```diff
@@ -252,6 +252,32 @@ var GridHelper = declare(null, {
     }
 });
 
+export var LocalStorage = dojo.declare([Evented], {
+    start: function () {
```

Comment:
Minor:  Rather than a separate "start" method, you could do this inside the constructor so it happens as part of the "new" call?